README.md

```diff
@@ -2,15 +2,14 @@
 
 # Project Name
 
-> To Do list
+> To Do list with javascript
 
 <!-- ![screenshot](./images/screenshot.png)
 ![screenshot](./images/screenshot1.png)
 ![screenshot](./images/contact-scrn.png)
 ![screenshot](./images/footer-scrn.png)
 ![screenshot](./images/footer-scrn.png) -->
 
-
 ## Built With
 
 - HTML
```

Comment:
- [ ] [Optional] Kindly, Add the full list of technology you used in this project for example ( HTML, JavaScript, React...etc).